BJS2-68375

tanufa25 opened the pull request:
User registration is added

Commit:
BJS2-68375
User registration is added

TheWandererr commented on src/main/java/school/faang/bjs2_68375/user_registration/Main.java:
```diff
@@ -0,0 +1,25 @@
+    private static void processUserCreationException(int index, String name, int age, String job, String address) {
+        try {
+            User user = new User(name, age, job, address);
+            System.out.println("user " + index + " is added: " + user);
```
Для сообщений использзуется Logger, в частности @slf4j

TheWandererr commented on src/main/java/school/faang/bjs2_68375/user_registration/User.java:
```diff
@@ -0,0 +1,41 @@
+    private final String address;
+
+    public User(String name, int age, String job, String address) {
+        if (name == null || name.isEmpty()) {
```
здесь и далее лучше использовать isBlank()

Commit:
BJS2-68375
@Slf4j is added for logging
isBlank() is used instead of isEmpty()

TheWandererr commented on src/main/java/school/faang/bjs2_68375/user_registration/Main.java:
```diff
@@ -0,0 +1,29 @@
+        processUserCreationException(11, "     ", 22, "Uber", "Amsterdam");
+    }
+
+    private static void processUserCreationException(int index, String name, int age, String job, String address) {
```
В целом, этот метод создает пользователя, именно это его целевое назвачение, то, что под капотом обрабатывается исключение - не имеет значение для потребителя этого метода, поэтому, название стоит сделать просто `createUser`. При этом метод должен "вернуть" нам созданного пользователя исходя из такого названия

Commit:
BJS2-68375
processUserCreationException is changed to createUser,
and User return statement is added in it.
